packages/fsapp/src/lib/app-router.ts

```diff
@@ -273,12 +274,12 @@ export default class AppRouter {
           Authorization: `Bearer ${this.cmsToken}`
         }
         // tslint:disable-next-line: cyclomatic-complexity
-      }).then(async (pageData: any) => {
+      }).then(async (pageData: AxiosResponse) => {
```

Comment:
I would personally prefer that types that can be inferred, like callback types from promises, to be inferred. This tends to feel more natural for me and allows `.get()` to use a generic to control this type.